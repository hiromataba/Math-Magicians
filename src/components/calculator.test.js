import React from 'react';
import renderer from 'react-test-renderer';
import Calculator from './calculator';

it('expect quote to match snapshot', () => {
  const tree = renderer.create(<Calculator />).toJSON();
  expect(tree).toMatchSnapshot();
});
