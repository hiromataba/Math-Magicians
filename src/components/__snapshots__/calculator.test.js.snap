// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect quote to match snapshot 1`] = `
<div
  className="calculator-page"
>
  <div
    className="welcome-txt"
  >
    <h2>
      Let us do some Math !
    </h2>
  </div>
  <div
    className="calculator"
  >
    <div
      className="result"
    >
      <input
        type="text"
        value="0"
      />
    </div>
    <div
      className="row-flex"
    >
      <button
        className="btn"
        onClick={[Function]}
        type="submit"
      >
        AC
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="submit"
      >
        +/-
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="submit"
      >
        %
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="submit"
      >
        รท
      </button>
    </div>
    <div
      className="row-flex"
    >
      <button
        className="btn"
        onClick={[Function]}
        type="submit"
      >
        7
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="submit"
      >
        8
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="submit"
      >
        9
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="submit"
      >
        x
      </button>
    </div>
    <div
      className="row-flex"
    >
      <button
        className="btn"
        onClick={[Function]}
        type="submit"
      >
        4
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="submit"
      >
        5
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="submit"
      >
        6
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="submit"
      >
        -
      </button>
    </div>
    <div
      className="row-flex"
    >
      <button
        className="btn"
        onClick={[Function]}
        type="submit"
      >
        1
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="submit"
      >
        2
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="submit"
      >
        3
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="submit"
      >
        +
      </button>
    </div>
    <div
      className="row-flex"
    >
      <button
        className="btn"
        onClick={[Function]}
        type="submit"
      >
        0
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="submit"
      >
        .
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="submit"
      >
        =
      </button>
    </div>
  </div>
</div>
`;
